subsys/bluetooth/common/Kconfig

```diff
@@ -209,9 +209,13 @@ config BT_HCI_VS
 	  Host and/or Controller. This enables Set Version Information,
 	  Supported Commands, Supported Features vendor commands.
 
+config BT_HCI_VS_FATAL_ERROR_SUPPORT
+	depends on BT_HCI_VS
```

Comment:
May want to remove this. Seems kconfig is not happy about selecting `BT_HCI_VS_FATAL_ERROR_SUPPORT` without enabling the support. And then keep the dependency on `BT_HCI_VS` in `BT_HCI_VS_FATAL_ERROR`.